assets/RNAseq_report.Rmd

```diff
@@ -459,11 +1179,12 @@ Here, volcano plots for all considered contrasts are shown. Hover over the dots
     log2FoldChange_min <- min(DE_all$log2FoldChange)
     log2FoldChange_max <- max(DE_all$log2FoldChange)
     pg <- ggplot(DE_all, aes(x=log2FoldChange, y=logpval, text=paste("Gene: ", gene_name, "<br>", "Log2FC: ", formatC(log2FoldChange, digits=2)))) +
-            geom_point(alpha=0.5, aes(color=ifelse(abs(DE_all$log2FoldChange) > params$log_FC &  DE_all$logpval > -log10(pval_threshold), "Differentially expressed genes", "Non-differentially expressed genes"))) +
+            #this goes after alpha=0.5:
+            geom_point(alpha=0.5, aes(color=ifelse(abs(DE_all$log2FoldChange) > params$log_FC_threshold &  DE_all$logpval > -log10(pval_threshold), "Differentially expressed genes", "Non-differentially expressed genes"))) +
             geom_hline(yintercept = 16, linetype= "dashed", size = 0.2, color = "grey") +
             geom_hline(yintercept = -log10(pval_threshold), size = 0.2, color = "grey") +
-            geom_vline(xintercept = log_FC_num, size = 0.2, color = "grey") +
-            geom_vline(xintercept = -log_FC_num, size = 0.2, color = "grey") +
+            geom_vline(xintercept = 0.05, size = 0.2, color = "grey") +
+            geom_vline(xintercept = -0.05, size = 0.2, color = "grey") +
```

Comment:
Why is this here `0.05` instead of the log_FC threshold?

Reply:
Ah yes, I already saw that as well. On it